Package.swift

```diff
@@ -76,7 +80,7 @@ let package = Package(
       name: "SwiftToolchain",
       dependencies: [
         .product(name: "AsyncHTTPClient", package: "async-http-client"),
-        .product(name: "SwiftToolsSupport-auto", package: "swift-tools-support-core"),
```

Comment:
This product dependency needs to be preserved, otherwise it causes linking issues on CI.

Reply:
I think it's a difference in resolution between 5.2 and 5.3. Without removing these lines it doesn't resolve on 5.3, and the 5.3 test fails for other reasons.

Reply:
Fair enough, should be a big problem after https://github.com/swiftwasm/carton/pull/195 is merged, where we have to drop support for 5.2.

Reply:
I don't have access to develop on 5.2 ATM and making guess PRs until it `works` is not great. I suggest we merge #195 and move on.

Reply:
Uh, I forgot about this bit. I can test with 5.2 on my side.

Reply:
I mean moving forward, since our dependency graph is large and there are (undocumented?) differences between the versions, plus 5.3 is stable and we have 5.4 to worry about - it's a good time to cut the chord IMO.

Reply:
Sure, it's the lack of output redirection with WAMR that I'm worried about. I'll try to get that sorted soon and we'll merge #195 after that.